### PR TITLE
[WIP] - Training i18n [Phuoc]

### PR DESCRIPTION
**Pull request by hua-phuoc-mulodo**
[training-i18n-phuoc] - Add i18n to posts

### CODE REVIEW

File: app/views/devise/registrations/edit.html.slim [hunk 1]
```diff
@@ -27,16 +27,16 @@ h2 Edit #{resource_name.to_s.humanize}
...
+    = f.submit t('devise.registrations.edit.update'), class: 'btn btn-primary'
 
-h3 Cancel my account
+h3 = t('devise.registrations.edit.cancel_my_account')
 
 p 
-  | Unhappy? 
-  = button_to("Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?" }, method: :delete)
+  | #{ t('devise.registrations.edit.unhappy') }? 
+  = button_to(t('devise.registrations.edit.cancel_my_account'), registration_path(resource_name), data: { confirm: "Are you sure?" }, method: :delete)
```
Comment (tran-hai-mulodo) [inline app/views/devise/registrations/edit.html.slim]: @hua-phuoc-mulodo 
We should translate for both Ruby and JavaScript code.
e.g: 
`t('devise.registrations.edit.are_you_sure')`

File: config/environments/development.rb [hunk 2]
```diff
@@ -52,4 +52,6 @@
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
   config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  config.i18n.available_locales = [:en, :ja, 'zh-CN', 'zh-TW']
+  config.i18n.default_locale = :ja
```
Comment (tran-hai-mulodo) [inline config/environments/development.rb]: @hua-phuoc-mulodo 
We can set the locale at the beginning of each request as below:
```
before_action :set_locale
def set_locale
  I18n.locale = params[:locale] || I18n.default_locale
end
```
Read here for more detail:  [2.2 Managing the Locale across Requests](http://guides.rubyonrails.org/i18n.html)

**Comment by hua-phuoc-mulodo** [comment]
I fix: data: { confirm: "Are you sure?" } to t('xx.yy.are_you_sure').

**Comment by tran-hai-mulodo** [comment]
@hua-phuoc-mulodo 
It's OK. I merged. Thanks.

**Pull request merged**